extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/DropTargetsSampler.java

```diff
@@ -8,25 +8,26 @@
 import io.opentelemetry.sdk.trace.data.LinkData;
 import io.opentelemetry.sdk.trace.samplers.Sampler;
 import io.opentelemetry.sdk.trace.samplers.SamplingResult;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
 
-public class DropNamesSampler implements Sampler {
+public class DropTargetsSampler implements Sampler {
     private final Sampler sampler;
-    private final List<String> dropNames;
+    private final List<String> dropTargets;
 
-    public DropNamesSampler(Sampler sampler, List<String> dropNames) {
+    public DropTargetsSampler(Sampler sampler, List<String> dropTargets) {
         this.sampler = sampler;
-        this.dropNames = dropNames;
+        this.dropTargets = dropTargets;
     }
 
     @Override
     public SamplingResult shouldSample(Context parentContext, String traceId, String name, SpanKind spanKind,
             Attributes attributes, List<LinkData> parentLinks) {
 
         if (spanKind.equals(SpanKind.SERVER)) {
-            for (String dropName : dropNames) {
-                if (name.startsWith(dropName)) {
-                    return SamplingResult.drop();
-                }
+            String target = attributes.get(SemanticAttributes.HTTP_TARGET);
+            // TODO - radcortez - Match /* endpoints
```

Comment:
@radcortez hmmm I just saw this TODO. Is it something that needs fixing soon?

Reply:
Not urgent, the previous implementation was working this way.